src/test/java/kitchenpos/domain/InMemoryOrderRepository.java

```diff
@@ -0,0 +1,39 @@
+package kitchenpos.domain;
+
+import java.util.*;
+
+public class InMemoryOrderRepository implements OrderRepository {
+    private Map<UUID, Order> orders = new HashMap<>();
+
+    @Override
+    public boolean existsByOrderTableAndStatusNot(OrderTable orderTable, OrderStatus status) {
+        return orders.values()
+                .stream()
+                .filter(order -> Objects.equals(order.getOrderTableId(), orderTable.getId()))
+                .filter(order -> !Objects.equals(order.getStatus(), status))
+                .findAny()
+                .isPresent();
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
                .anyMatch(order -> order.getOrderTableId().equals(orderTable.getId()) && order.getStatus() != status);
```

Reply:
anyMatch를 사용하니 좀더 간결해 지네요
피드백주신 내용으로 개선해보았습니다.